bitnami/concourse/templates/NOTES.txt

```diff
@@ -2,9 +2,14 @@ CHART NAME: {{ .Chart.Name }}
 CHART VERSION: {{ .Chart.Version }}
 APP VERSION: {{ .Chart.AppVersion }}
 
+{{- $releaseNamespace := .Release.Namespace }}
+{{- $clusterDomain := .Values.clusterDomain }}
+{{- $fullname := include "concourse.web.fullname" . }}
+{{- $concourseWebSecretName := include "concourse.web.secretName" . }}
+
```

Comment:
As there is diagnosticMode support, I would add the special notes when the diagnostic mode is enabled. Checke other charts like PostgreSQL